db/migrate/20190708105607_create_audit_logs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAuditLogs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :audit_logs do |t|
+      t.integer :user_id
```

Comment:
It'd be great if this table could be referenced to the user tables, with a foreign key.

Basically `t.references :user, foreign_key: true` instead of `t.integer :user_id`

Reply:
Great note, @rhymes This will also add an index which doesn't exist in the original migration.

Reply:
Great point. I'll fix this